content/docs/1.7.0/deploy/important-notes/index.md

```diff
@@ -30,6 +31,12 @@ Please see [here](https://github.com/longhorn/longhorn/releases/tag/v{{< current
   - [Snapshot Creation Time as shown in the UI occasionally changes](#snapshot-creation-time-as-shown-in-the-ui-occasionally-changes)
   - [Unable to Revert a Volume to a Snapshot Created before Longhorn v1.7.0](#unable-to-revert-a-volume-to-a-snapshot-created-before-longhorn-v170)
 
+## Deprecation
+
+- The `environment_check.sh` script  
+  Since Longhorn v1.7.0, we introduce the [Longhorn Command-line Tool](#introduction-of-commandline-tool). The functionalities of the script `environment_check.sh` will overlap
+with the Longhorn Command-line Tool. Therefore the script `environment_check.sh` is deprecated at v1.7.0 and scheduled for removal at v1.8.0.
+
```

Comment:
```suggestion
## Deprecation

### Environment Check Script

```suggestion
  Since Longhorn v1.7.0, we have introduced the [Longhorn Command line Tool](../../advanced-resources/longhornctl/). The functionality of the environment check script, [environment_check.sh](https://github.com/longhorn/longhorn/blob/master/scripts/environment_check.sh) overlaps with that of the Longhorn Command-line Tool. Therefore, the script has been deprecated as of v1.7.0 and is scheduled for removal in v1.8.0.
```

Also add linkof

Reply:
Modified. Thanks